application/src/Controller/MatrixController.php

```diff
@@ -310,6 +310,12 @@ public function roomState(string $serverID, string $roomID, string $eventType, R
                 return $check['message'];
             }
             $room->setAvatar($payload->url);
+        } elseif ($eventType == 'm.room.power_levels') {
+            // Update room avatar.
+            $check = $this->validateRequest((array)$payload, ['users']);
+            if (!$check['status']) {
+                return $check['message'];
+            }
```

Comment:
There isn't any update call. I'd expect something here, or a comment to explain that we aren't doing it?

Reply:
None of the state api's in mock is setting anything, they are just validating the request that the required data in the payload is available. Updated the comment block to explain more.